### DISCUSSION TITLE
DDNS Client with lego as base

### DISCUSSION THREAD

**Discussion by mietzen** [go-acme/lego#1888] [Ideas]
Hi, 
I post this in "Ideas" because I think that this is to much out of scope for this project, but I would like to hear your opinion on this: 

I stumbled over lego when searching for alternatives of [DDClient](https://github.com/ddclient/ddclient), since DDClient has trouble finding new maintainers and possibly won't be maintained in the future (see: https://github.com/ddclient/ddclient/issues/528).

From briefly looking at legos code and functionality I would think that +90% of what we need for DDNS-Client is already available. 
What is your opinion on that? How much work would one have to invest to turn (parts of) lego into a DDNS-Client.

Cheers

**Comment by ldez** [comment]
Hi,

We don't plan to become a universal lib for DNS.
We focus only on adding/removing TXT records.
We don't handle updates of IPs for a dynamic domain.

**Comment by mietzen** [comment]
As said above, I agree that this wouldn’t be part of lego. This would be a independent project with parts of lego as base.

My general question was, if using lego as basis would be a good or a terrible idea.
